influxql/parser.go

```diff
@@ -1476,6 +1476,11 @@ func (p *Parser) parseField() (*Field, error) {
 	if err != nil {
 		return nil, err
 	}
```

Comment:
Thought about it, but the invalid expression might be deep into the field. Example:

`select value * (2 > 3) * 43 from foo`

The position information will be lost by the time that the parser yields the expression at the end of the 43 token.